app/models/vm_or_template/operations/configuration.rb

```diff
@@ -129,6 +129,16 @@ def detach_volume(volume_id, _device = nil)
     raw_detach_volume(volume_id)
   end
 
+  def raw_clone_volume(_options = {})
+    raise NotImplementedError, _("must be implemented in a subclass")
+  end
+
+  def clone_volume(_options = {})
+    raise _("VM has no EMS, unable to clone volume") unless ext_management_system
+
+    raw_clone_volume(_options = {})
```

Comment:
The leading `_` is a hint to the linter (rubocop) that the variable is unused but that is expected.  If you're using the variable it should not have a leading `_`
```suggestion
  def clone_volume(options = {})
    raise _("VM has no EMS, unable to clone volume") unless ext_management_system

    raw_clone_volume(options = {})
```